### PR TITLE
update docs for debian testing and unstable

### PR DESCRIPTION
**Pull request by lsm5**
Buildah is now present on debian testing and unstable so providing Kubic
repos for it can lead to conflicts. Better to mention only the official
repos to avoid confusing end users.
Related: https://github.com/containers/buildah/issues/2797

Also, provide installation steps for Ubuntu 20.10 and newer, and redirect
Raspberry Pi OS users to Debian installation steps to
avoid content duplication.

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

docs change for debian installation

#### How to verify it

try the steps out if necessary

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
None

#### Does this PR introduce a user-facing change?

```release-note
None
```

@rhatdan @TomSweeneyRedHat @vrothberg @EvanCarroll PTAL

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM
@vrothberg @TomSweeneyRedHat PTAL

File: install.md [hunk 1]
```diff
@@ -59,35 +59,31 @@ sudo dnf -y install buildah
...
+The [Kubic project](https://build.opensuse.org/project/show/devel:kubic:libcontainers:stable)
+provides packages for Debian 10. The Kubic packages for Debian Testing/Bullseye and Debian Unstable/Sid
+have been discontinued to avoid
+[conflicts](https://github.com/containers/buildah/issues/2797) with the official packages.
 
-# Debian 10
-echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Debian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
-wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Debian_10/Release.key -O Release.key
+Caution: If you upgrade from Debian 10 to Testing/Bullseye or
+Unstable/Sid you would likely end up downgrading buildah because the version in
```
Comment (TomSweeneyRedHat) [inline install.md:78]:
```suggestion
Unstable/Sid you would likely end up downgrading Buildah because the version in
```

**Comment by TomSweeneyRedHat** [comment]
one nit, otherwise LGTM

**Comment by rhatdan** [comment]
Saving the planet and merging.

**Pull request merged**